### ISSUE TITLE
Add timestamp as document footer

### ISSUE THREAD

**Issue by toderash** [aspirepress/documentation#40]
I would like to include a timestamp on our documents, such as "This document was last modified on November 16, 2024 at 05:00 UTC."

This should be don for each file in the documentation repo as well as the governance repo and the interop spec repo... basically all the document files (.md etc, as opposed to code).

The date should be appear as a footer below a horizontal line, and be derived from the file timestamp or date/time that it is merged.

Some latitude is given for how this is best to implement. Automating it would be best, though a shortcode of sorts that gets parsed when committed would work, just relies on an extra step. What I'm looking for is a notice that would not only show up on Github, but also appear when the document appears at docs.aspirepress.org so the latest version date is open and transparent without anyone having to navigate gh prs to find it.

**Comment by asirota** [comment]
This is an implementation

https://github.com/just-the-docs/just-the-docs/discussions/711

**Comment by toderash** [comment]
I like the approach of a global footer to insert the timestamp, as it lets us easily add other material like the license info and links. The examples shown offer configuration of date and time format, but I didn't see timezone. If that's there and just not in the example, then this works for me. (Commenting on the footer implementation, not the whole theme.)

**Comment by AmnestyAM** [comment]
yeah it outputs the HTML but nothing inside, My guess is it needs the front matter, and then overrides it. I will set up a Jekyll site locally on the weekend, so I can test it more thoroughly

**Comment by asirota** [comment]
front matter example for _config.yml

last_edit_timestamp: true # show or hide edit time - page must have `last_modified_date` defined in the frontmatter
last_edit_time_format: "%b %e %Y at %I:%M %p" # uses ruby's time format: https://ruby-doc.org/stdlib-2.7.0/libdoc/time/rdoc/Time.html```
